src/sections/Home/Banner/index.js

```diff
@@ -1,26 +1,24 @@
 import React, { useEffect, useState } from "react";
 import BannerDefault from "../Banner-default";
-// import BannerAlt from "../Banner-alt"; 
+import BannerAlt from "../Banner-alt"; 
 
 const Banners = [
   <BannerDefault key={0} />,
-  // <BannerAlt key={1}/> 
+  <BannerAlt key={1}/> 
 ];
 
 const Banner = () => {
   const [Banner, setBanner] = useState(0);
   useEffect(() => {
     if (sessionStorage.getItem("banner")) {
-      sessionStorage.setItem("banner", (1 + Number(Banner)) % Banners.length);
+      sessionStorage.setItem("banner", (1 ^ Number(Banner)));
     } else {
       sessionStorage.setItem("banner", 0);
     }
-  });
-  useEffect(() => {
     if (Banner != sessionStorage.getItem("banner")) {
       setBanner(sessionStorage.getItem("banner"));
     }
-  });
+  }, []);
```

Comment:
Here, when we go back to home page for the 3rd time, it won't render `Banner-default` but `Banner-alt` which is not desired. Looking into it! 🔍